xarray/core/ops.py

```diff
@@ -520,24 +522,40 @@ def inject_bottleneck_rolling_methods(cls):
     for name, f in methods:
         func = cls._reduce_method(f)
         func.__name__ = name
-        func.__doc__ = _ROLLING_REDUCE_DOCSTRING_TEMPLATE.format(name=func.__name__)
+        func.__doc__ = _ROLLING_REDUCE_DOCSTRING_TEMPLATE.format(
+            name=func.__name__)
         setattr(cls, name, func)
 
     # bottleneck rolling methods
     if has_bottleneck:
-        if LooseVersion(bn.__version__) < LooseVersion('1.0'):
+        bn_version = LooseVersion(bn.__version__)
```

Comment:
Should we move the minimum bottleneck version to 1.1 and just get rid of all this then?